exporter/elasticsearchexporter/README.md

```diff
@@ -16,13 +16,75 @@ This exporter supports sending OpenTelemetry logs and traces to [Elasticsearch](
 
 ## Configuration options
 
-- `endpoints`: List of Elasticsearch URLs. If `endpoints` and `cloudid` are missing, the
-  ELASTICSEARCH_URL environment variable will be used.
-- `cloudid` (optional):
-  [ID](https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html) of the
-  Elastic Cloud Cluster to publish events to. The `cloudid` can be used instead
-  of `endpoints`.
-- `num_workers` (default=runtime.NumCPU()): Number of workers publishing bulk requests concurrently.
+Exactly one of the following settings is required:
+
+- `endpoint` (no default): The target Elasticsearch URL to which data will be sent
+  (e.g. `https://elasticsearch:9200`)
+- `endpoints` (no default): A list of Elasticsearch URLs to which data will be sent
+- `cloudid` (no default): The [Elastic Cloud ID](https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html)
+  of the Elastic Cloud Cluster to which data will be sent (e.g. `foo:YmFyLmNsb3VkLmVzLmlvJGFiYzEyMyRkZWY0NTY=`)
+
+If none of the above attributes are specified, one or more Elasticsearch URLs
```

Comment:
> ^^ p.s. I'm more used to env variables overriding, so even when they are set, they would be ignored on env. If that's the case in the logic, we need to change the docs accordingly.

I am too, but that's not how it works at the moment. This behaviour comes from go-elasticsearch: https://github.com/elastic/go-elasticsearch/blob/de21bec3133f09d8821c3a57c2782d25417fdfed/elasticsearch.go#L171-L175

IMO we should get rid of this in a future change, and expect users to use standard OTel Collector mechanisms for overriding config: https://opentelemetry.io/docs/collector/configuration/#override-settings

Reply:
> Since I suggested to inline the round-robin part into the attribute desc, if that makes sense to do, we can remove the repeat about round-robin

:+1: will do